src/obs-websocket.h

```diff
@@ -19,6 +19,115 @@ with this program. If not, see <https://www.gnu.org/licenses/>
 
 #pragma once
 
+/**
```

Comment:
I think it would be best to define these individually wherever these objects are most created, be it things like the utils code files or request/event files. Just to align with how documentation is currently done for these.

Reply:
I moved them around, let me know if what I did aligns with what you were thinking or if there's even better places for them.